_component_design/breadcrumbs.md

```diff
@@ -66,20 +63,20 @@ Sometimes we can get so deep in an interface we lose track of where we are. Whil
 {% endcolumn %}
 </div>
 
-
 <div class="row">
 {% column left:"col-md-4" %}
 ## To many items
 
-* __Jump The Crumb:__ Maybe we’ve really overdone it on the architecture. It happens (does it have to though?). Whattaya do? If you can’t abbreviate adequately, we have to jump the breadcrumbs. What’s that mean?
+<!---* __Jump the crumb:__ Maybe we’ve really overdone it on the architecture. It happens (does it have to though?). Whattaya do? If you can’t abbreviate adequately, we have to jump the breadcrumbs. What’s that mean?--->
```

Comment:
@technabors - I have commented this 'jump the crumb' thing out because I have absolutely 0 idea what this means.

Reply:
That is just a pithy short hand that Roman came up with. I'm fine with removing it. 

how about the following alt text - 

Truncated breadcrumbs: While it should be avoided, there are situations applications architecture creates a situation where breadcrumbs surpass four items. If instances such as this, either of the following alternate patterns can be used.

Reply:
We can either remove this section has you have done, or change it to a `<p> `with the above proposed alt text and then leave the two following sections, "what to keep" && "Still to long" as `<li>`. Up to your discretion.